src/multi-selector.css

```diff
@@ -32,6 +32,11 @@
   overflow-y: auto;
 }
 
+/* Do not remove. This class is targeted with JS */
```

Comment:
Why would removing the css rule effect targeting with JS?

Reply:
This specific CSS class is being targeted

Reply:
https://github.com/CanopyTax/cpr-multiselect/blob/a5d135246a6784f967f1e23091713805cf57110f/src/multi-selector.js#L414

This is the line that needs this CSS rule